### DISCUSSION TITLE
"user not found"

### DISCUSSION THREAD

**Discussion by Tekgnome-svg** [authelia/authelia#4226] [Q&A] [answered]
edit resolved.
I'm a goober and was editing user_databse.yml not user_database.yml

Original post:

Deployed the latest version via Portainer

I'm not sure where I went wrong but it's like Authelia isn't reading the content inside of the users_database.yml file. I get the same "user not found" error on any of the three users I have listed. I have restarted the container multiple times with no effect. 

Portainer Logs
![image](https://user-images.githubusercontent.com/77984415/197226660-15756ec7-1323-4b18-8232-d1d4f1d6c624.png)

user_database.yml
![image](https://user-images.githubusercontent.com/77984415/197224374-b5f4a419-7934-4bc7-a34a-22137aba9bc1.png)

Configuration.yml
![image](https://user-images.githubusercontent.com/77984415/197227740-15a7e435-1640-4a73-9f5a-8e5239c867ba.png)

**Comment by james-d-elliott** [comment, marked as answer]
Answered by user in OP.
